README.md

```diff
@@ -1,2 +1,94 @@
 # npmx-zephyr
-npmx integration with Zephyr
+
+<p align="center">
+    <img src="doc/_static/images/npm_family_logo_background.png" alt="NPMX LOGO" width="300" height="300">
+</p>
+
+## Overview
+npmx-zephyr repository provides a convenient way to work with Nordic Semiconductor's Power Management ICs (PMICs) by integrating [npmx library][npmx_repo] into the [Zephyr][zephyr_site] RTOS environment.
```

Comment:
```suggestion
npmx-zephyr repository provides an example of [npmx library][npmx_repo] integration with the [Zephyr][zephyr_site] RTOS environment. This can be used as a starting point for an integration with other RTOS or bare-metal environment.
```

Reply:
... with other RTOS or bare-metal environments.
or
... with another RTOS or bare-metal environment.
?
@masz-nordic

Reply:
For now I'll go with `other` option